src/Umbraco.Web.UI.Client/src/packages/tags/components/tags-input/tags-input.element.ts

```diff
@@ -208,9 +218,9 @@ export class UmbTagsInputElement extends UUIFormControlMixin(UmbLitElement, '')
 	}
 
 	#enteredTags() {
-		return html` ${this.items.map((tag) => {
+		return html` ${this.items.map((tag, index) => {
```

Comment:
Would be nice to use Lit Repeat instead of Map. Repeat should be more performant :-)